torch/_inductor/triton_heuristics.py

```diff
@@ -339,6 +339,7 @@ def _precompile_config(self, cfg: Config, warm_cache_only_with_cc: Optional[int]
                 "num_warps": compile_meta["num_warps"],
                 "num_stages": compile_meta["num_stages"],
                 "debug": compile_meta["debug"],
+                "enable_fp_fusion": False,
```

Comment:
This is to disable FMA?

I'm wondering if this should be in a separate PR even though it's just a single line. I know that you recently posted about an numerical issue caused by FMA. But on the other hand, theoretically speaking disabling FMA may have perf hit due to more computation instructions. Do we observe any perf impact?

Reply:
Yeah I'll put it in a separate PR - @peterbell10 is working on that.